### DISCUSSION TITLE
Create and Edit object pattern and component usage

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#1086] [Office hours] [answered]
Hello

Greatly appreciate feedback on the recommended pattern for Create as well as editing the existing object? 

We are trying to move existing Fleet pages, Create and Update pages to Paste components. And like to get Feedback on component usage and interaction of overall IoT-Fleet create and edit pages.

Specific questions
**Create Object - Create Fleet Page**
1. Accuracy of Component usage
2. Grid or layout of the page. 
3. tool tip vs. supporting text for create page
4. Best/Consistent/accessible friendly method to show dependencies of form fields. If Data service is enabled allow user to configure IP Command Callback URL. If Data service is not enabled hide IP Command Callback URL.

Edit Object -- Edit existing Fleet Page
1. Accuracy of Component usage
2. Grid or layout of the page. 
3. Page title (Generic name vs. object name)
4. tool tip vs. supporting text for edit page

![create-edit-fleet-objects](https://user-images.githubusercontent.com/69016180/104221305-92356d80-53f5-11eb-854b-117a12478998.png)

![discussions-1086-2](https://user-images.githubusercontent.com/69016180/104271914-d7d35400-5450-11eb-8016-7562d008b5ea.png)

**Comment by corinne-nagel** [comment, marked as answer]
Hi @ishah-twilio we've reviewed your designs and left [feedback in Figma](https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Ishita-1086-Create-Edit-Patterns?node-id=0%3A1). We suggest you reach out to Jo and Caroline about the designs as well. Let us know if the feedback was unclear or didn't make sense. Thanks!

**Reply by ishah-twilio** [reply]
@corinne-nagel - thank you. How do i get Figma access?

**Reply by TheSisb** [reply]
Looks like you received Figma access now. I've gone ahead and marked this Discussion resolved, feel free to comment if anything else pops up.

**Comment by ishah-twilio** [comment]
Hello team -- 

During office hour -- I am still hoping to get feedback on showing dependencies of form fields (see attached screen for more details).

![discussions-1086-2 - V3](https://user-images.githubusercontent.com/69016180/106192757-7655f980-6161-11eb-9334-54e2d3450ce0.png)
